src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,187 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
```

Comment:
Could you add a keywords list also? See https://github.com/mongodb/genny/blob/73acba4a6866587e78f32d7b0261e9a1ca6f04c0/docs/using.md#L153


 For example https://github.com/mongodb/genny/blob/73acba4a6866587e78f32d7b0261e9a1ca6f04c0/src/workloads/scale/BigUpdate.yml#L9